srcpkgs/dpdk/template

```diff
@@ -53,7 +51,7 @@ dpdk-devel_package() {
 		vmove usr/lib/dpdk/pmds-${version}/*.so
 		vmove usr/lib/pkgconfig
 		vmove usr/share/dpdk/examples
-	}
+}
```

Comment:
Accident? The old indentation was correct.

Reply:
Probably :)